ntrans.py

```diff
@@ -1,107 +1,75 @@
-import nltk
 import translatepy
-import re
-import collections
 import csv
-# import ssl
 
+"""
+DEVELOPER INFO:
```

Comment:
People reading this are developers. They know it. There's no need to tell that to them.

Also, use `#` instead of `"""` for comments. When you see `"""` somewhere, those are usually **not** comments, but docstrings. I have written more about docstrings in my python-tutorial. This is not a valid docstring, because it is after the imports; a docstring must be the first thing in the file or function or class it's documenting.